prompt/prompt-structure/src/commonTest/kotlin/ai/koog/prompt/structure/json/generator/JsonSchemaGeneratorTest.kt

```diff
@@ -162,6 +167,181 @@ class JsonSchemaGeneratorTest {
         val recursiveProperty: RecursiveTestClass?
     )
 
+
+    @Serializable
+    @SerialName("IntMinTest")
+    data class IntMinTest(
+        @property:LLMMin(5) val min: Int
+    )
+
+    @Serializable
+    @SerialName("IntMaxTest")
+    data class IntMaxTest(
+        @property:LLMMax(5) val max: Int
+    )
+
+    @Serializable
+    @SerialName("IntRangeTest")
+    data class IntRangeTest(
+        @property:LLMRange(5, 10) val range: Int
+    )
+
+    @Serializable
+    @SerialName("IntDuplicateAnnotationTest")
+    data class IntDuplicateAnnotationTest(
+        @property:LLMMax(5)
+        @property:LLMRange(7, 9)
+        val stringProperty: Int
+    )
+
+
+    @Serializable
+    @SerialName("StringMinTest")
+    data class StringMinTest(
+        @property:LLMMin(5) val min: String
+    )
+
+    @Serializable
+    @SerialName("StringMaxTest")
+    data class StringMaxTest(
+        @property:LLMMax(5) val max: String
+    )
+
+    @Serializable
+    @SerialName("StringRangeTest")
+    data class StringRangeTest(
+        @property:LLMRange(5, 10) val range: String
+    )
+
+    @Serializable
+    @SerialName("StringDuplicateAnnotationTest")
+    data class StringDuplicateAnnotationTest(
+        @property:LLMMax(5)
+        @property:LLMRange(7, 9)
+        val stringProperty: String
+    )
+
+
+    @Serializable
+    @SerialName("ListMinTest")
+    data class ListMinTest(
+        @property:LLMMin(5) val array: List<String>
+    )
+
+    @Serializable
+    @SerialName("ListMaxTest")
+    data class ListMaxTest(
+        @property:LLMMax(5) val array: List<String>
+    )
+
+    @Serializable
+    @SerialName("ListRangeTest")
+    data class ListRangeTest(
+        @property:LLMRange(5, 10) val array: List<String>
+    )
+
+    @Serializable
+    @SerialName("ListDuplicateAnnotationTest")
+    data class ListDuplicateAnnotationTest(
+        @property:LLMMax(5)
+        @property:LLMRange(7, 9)
+        val array: List<String>
+    )
+
+
+    @Test
+    fun testIntMin() {
+        val result = basicGenerator.generate(json, "IntMinTest", serializer<IntMinTest>(), emptyMap())
+        val schema = json.encodeToString(result.schema)
+
+        assertContains(schema, "\"minimum\": 5")
```

Comment:
I would prefer a full resulting JSON check in asserts with `assertEquals`, not just contains. Just to be sure and to make it more visible, what the final result is supposed to look like